app/src/main/kotlin/github/com/rhacco/dota2androidapp/activities/LiveMatchesActivity.kt

```diff
@@ -29,21 +39,26 @@ class LiveMatchesActivity : BaseLifecycleActivity<MatchesViewModel>() {
     }
 
     // Add tournament matches first, then sort by average MMR (descending)
-    private fun updateLiveMatchesListData(listItemData: LiveMatchesListAdapter.ListItemData) {
-        if (listItemData.mAverageMMR < 1) {
-            mListAdapter.mListItemsData.add(0, listItemData)
+    private fun updateLiveMatchesListData(newItem: LiveMatchesListAdapter.ListItemData) {
+        // Strangely, sometimes the same match arrives here twice, so prevent creating duplicates
```

Comment:
Uhhh...... "I dont know" is not a good sign this early into development. Please find the reason why.

Reply:
No longer relevant